src/styles/_catalog.scss

```
@@ -1,8 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 48px;
 
-  max-width: 944px;
+  max-width: 200px;
+
+  @media (min-width: 488px) {
+    max-width: 448px;
+  }
```

You code is working good but you can use  another possible way without hardcoding width
```suggestion
  @media (min-width: 488px) {
   grid-template-columns: repeat(2, $CardWidth);
  }
```